src/gui/torrentcontenttreeview.h

```diff
@@ -46,6 +46,7 @@ class TorrentContentTreeView final : public QTreeView
     explicit TorrentContentTreeView(QWidget *parent = nullptr);
     void keyPressEvent(QKeyEvent *event) override;
 
+    void setupDownloadPriorityMenu(QMenu *menu, bool createSubMenu) const;
```

Comment:
you should add forward declaration:
```c++
#include <QTreeView>

class QMenu;
```

Reply:
Fixed by a21714412a4cdf1b86d8ea480bcb160c475358a4